srcpkgs/xone/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'xone'
+pkgname=xone
+version=0.3
+revision=1
+depends="curl cabextract dkms"
+short_desc="Modern Linux driver for Xbox One and Xbox Series X|S controllers"
+maintainer="xolophreny <xolophreny@protonmail.ch>"
+license="GPL-2.0-only"
+homepage="https://github.com/medusalix/xone"
+distfiles="https://github.com/medusalix/xone/archive/refs/tags/v${version}.tar.gz"
+checksum=993f6b2b07c3236ce283d5de4da47dbfc16a86e056af504a4958d87f718ece20
+
+dkms_modules="${pkgname} ${version}"
+
+do_install() {
+	vinstall install/modprobe.conf 644 /usr/lib/modprobe.d xone.conf
+	vbin install/firmware.sh xone-update-firmware
+	find . -type f \( -name dkms.conf -o -name '*.c' \) -exec sed -i "s/#VERSION#/$version/" {} +
+	echo 'ccflags-y += -DDEBUG' >> ./Kbuild
+	vmkdir /usr/src/${pkgname}-${version}
+	vcopy . /usr/src/${pkgname}-${version}
```

Comment:
this installs some extra files

Reply:
Should be cleaner now, although not sure if that's the way to do it

Reply:
that's how most dkms packages do it, I think